packages/labs/analyzer/src/lib/javascript/classes.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Naming the containing folder "javascript" seems a little obtuse. Can you describe the contents more?

Reply:
I might name it "typescript" - but the context is analysis utilities for TS, as opposed to possibly future utilities for custom-elements-manifest, CSS, or HTML.